tools/build_rules/verifier_test/rust_indexer_test.bzl

```diff
@@ -88,8 +88,9 @@ rust_extract = rule(
         "crate_name": attr.string(
             default = "test_crate",
         ),
-        "_extractor": attr.label(
+        "extractor": attr.label(
```

Comment:
Having an executable that is core to the rule's function be user-specified is often unwise; what's the use case you have envisioned here?